src/Components/Collections/CollectionsUtils/TypesEnumsUtils/UsedByType.js

```diff
@@ -0,0 +1,66 @@
+import {makeStyles} from '@material-ui/core/styles';
+import {withVlow} from 'vlow';
+import Chip from '@material-ui/core/Chip';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {TypeStore} from '../../../../Stores';
+
+const withStores = withVlow([{
+    store: TypeStore,
+    keys: ['customTypes']
+}]);
+
+const useStyles = makeStyles(theme => ({
+    chips: {
+        margin: theme.spacing(1),
+    },
+}));
+
+const UsedByType = ({customTypes, name, onChangeItem, scope}) => {
+    const classes = useStyles();
+    const u = customTypes[scope]?customTypes[scope].filter(i=>
+        `${i.fields},`.includes(`,${name},`) ||
```

Comment:
This list of checks is a bit confusing to me, Isn't this easier and more easy to maintain using a regular expression? For example:
```
.*([A-Za-z_][0-9A-Za-z_]{0,254}).*
```

and then just take the first capture group?

Reply:
Yes I will look into it. Thanks!